test/language/class/class-super-in-static-setter.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 14.5
+description: >
+    class super in static setter
+---*/
+class B {
+  static method() {
+    return 1;
+  }
+  static get x() {
+    return 2;
+  }
+}
+class C extends B {
+  static set x(v) {
+    assert.sameValue(v, 3);
+    assert.sameValue(super.x, 2);
+    assert.sameValue(super.method(), 1);
```

Comment:
Do you think it's worth added a flag to avoid false positives (i.e. if the setter is never called)?